api/controllers/DashboardController.js

```diff
@@ -7,15 +7,26 @@
 
 const validate = require('../hooks/validate/validator');
```

Comment:
@CalvinRodo : I probably should have done some of the refactors in this file in a separate PR, in hindsight. Would love your review on this file in case I did anything too wildly impacting. (Also, fixed a small bug on `supportingDocuments` not being the same name in a place or two, such as `supportingDocs` in one spot -- in case you're seeing that bug elsewhere.)